XS✔ ◾ rewrote rule on keeping devs away from design work (#9229)
SSWConsulting/SSW.Rules.Content: rules/keep-developers-away-from-design-work/rule.md

---
seoDescription: Learn why separating design tasks from development roles is crucial for maintaining a unified product vision and avoiding costly UX Debt.
type: rule
archivedreason:
title: Do you keep developers away from tasks involving design or the overall look and feel?
guid: 6a276fd5-bff9-4ebf-ae2c-93f03647dd42
uri: keep-developers-away-from-design-work
created: 2015-10-07T03:50:51.0000000Z
authors: 
  - title: Tiago Araujo
    url: https://ssw.com.au/people/tiago-araujo
  - title: Jayden Alchin
    url: https://www.ssw.com.au/people/jayden-alchin
  - title: Betty Bondoc
    url: https://www.ssw.com.au/people/betty-bondoc
  - title: Adam Cogan
    url: https://ssw.com.au/people/adam-cogan
related: []
redirects:
  - do-you-keep-developers-away-from-anything-related-to-design-or-look-and-feel
---

> It’s tempting for companies to blur the lines between design and development roles, especially when resources are limited.  Maintaining a clear distinction between disciplines is crucial for creating cohesive and effective user experiences.

Reviewer: ```suggestion
When resources are limited, companies often blur the lines between design and development roles in an attempt to maximize efficiency. However, maintaining a clear distinction between disciplines is crucial for creating cohesive and effective user experiences.
```

For most developers, especially those early in their careers, the risk of making choices that negatively impact the overall design is too high. When Product Owners delegate design work to developers, they are likely to accumulate accumulate significant UX Debt.

<!--endintro-->

::: bad  
![Figure: Bad design can be dangerous!](BadDesignGun.jpg)  
:::

> While it’s important to be flexible in real-world situations, particularly when dealing with a low designer-to-developer ratio, it's still wise to keep design work primarily in the hands of designers.

Reviewer: ```suggestion
While flexibility is important, particularly when dealing with a low designer-to-developer ratio, it's still wise to keep design work primarily in the hands of designers to maintain consistency and quality.
```

## Streamlining design decisions

Design is an inherently subjective field, and even experienced designers don't always agree on every decision.

> Design needs to be approached holistically, meaning every element should be considered as part of the whole. For example, color choices, typography, and user experience should all work together to create a unified design. When too many people, especially those without a design background, get involved, it can result in a disjointed and inconsistent product.

Reviewer: ```suggestion
In addition, effective design demands a holistic approach, where layout, typography, color, UX, and other essential elements work together to create cohesion. When non-designers, especially those without a strong grasp of these principles, get involved, the result is often disjointed and inconsistent, leading to a fragmented user experience.
```
